todoCLI.js

```diff
@@ -0,0 +1,140 @@
+//Initializing the readline and filesystem modules
+const readline = require('readline');
+const fs = require('fs');
+const user = readline.createInterface(process.stdin, process.stdout);
+//Initializing a todo list as an empty array
+let todo = [];
+//Initializing a string for file writing
+let string = '';
+
+//Function that checks if todo is empty and otherwise prints the contents of the todo list
```

Comment:
your code is already saying what you said in the comment. don't need to repeat it.